source/supported-hardware/upxtremei12.rst

```diff
@@ -90,6 +90,54 @@ Stitch |SPN| images with factory BIOS image using the stitch tool::
 
 See :ref:`stitch-tool` on how to stitch the IFWI image with |SPN|.
 
+Slimbootloader binary for capsule
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+A capsule image could ecapsulate a Slimbootloader image to be used in firmware update mechanism. More information is described in :ref:`firmware-update`.
+
+For the process running properly, an IFWI image programmed on board with TOP SWAP size configuration is required.
+
+.. note:: Enabling IFWI capsule update feature requires additional firmware components and tools.
```

Comment:
Can you clarify what is referred to as IFWI capsule update ?

Reply:
@rprangar here IFWI capsule update feature is referring to TOP SWAP size setting,  To be specific maybe just re-write as TOP SWAP size

Reply:
@rprangar edited, pls check again.